src/contracts/proxies/HaiSafeManager.sol

```diff
@@ -120,12 +125,16 @@ contract HaiSafeManager {
     _usrSafes[_usr].add(_safeId);
     _usrSafesPerCollat[_usr][_cType].add(_safeId);
 
+    IVault721(vault721).mint(_usr, _safeId);
```

Comment:
storing the object instead of storing the address for vault721 should save gas for users

Reply:
👍🏼